### PR TITLE
Add the build python package

### PR DESCRIPTION
**Pull request by jasongrout**
This adds the Python [build](https://pypi.org/project/build/) package as a conda-forge package (https://github.com/pypa/build). This package, from the Python Packaging Authority team, is used to build python sdist and wheel packages.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jasongrout** [comment]
@FFY00, @gaborbernat: I'm trying to be helpful here in packaging up `build` as a conda-forge package. We do a lot of our release processes in Jupyter with conda environments, so it would be really helpful to have build as a conda-forge package (since we're slowly moving to using `build` at @gaborbernat's suggestion).

I put myself as a maintainer to try to be helpful. I'm also happy to delete myself and leave it to you and whoever else you'd like to include.

If you'd like to be listed as a maintainer for this conda-forge package, please confirm that you accept.

**Comment by jasongrout** [comment]
The artifacts produced by the test builds here are at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=289113&view=artifacts&pathAsName=false&type=publishedArtifacts

File: recipes/build/meta.yaml
Comment (FFY00) [inline recipes/build/meta.yaml:33]: Missing `build.env`.

**Comment by FFY00** [comment]
That's fine :+1:

**Review by gaborbernat — APPROVED** [review]
Yeah, please add me too. Thanks!

File: recipes/build/meta.yaml
Comment (jasongrout) [reply recipes/build/meta.yaml:33]: Fixed

**Comment by jasongrout** [comment]
Setting as ready for review, then. Thanks!

**Comment by jasongrout** [comment]
Ready for review, @conda-forge/help-python. Thanks!

**Review by dopplershift — APPROVED** [review]

**Comment by beckermr** [comment]
It turns out this PR was a dupe of python-build which was already on conda-forge. I am going to archive the build-feedstock.

**Comment by beckermr** [comment]
For posterity, you can install `python-build` to get this package.

**Comment by beckermr** [comment]
@conda-forge/build if we mark all of the `build` packages a broken, will that break too much of your stuff using the `build` conda package?

FYI, once a package is marked broken it will no longer be installable in new environments and so CI jobs and other envs that depend on being able to install `build` will fail. 

cc @isuruf

**Comment by beckermr** [comment]
This convo has moved to https://github.com/conda-forge/build-feedstock/issues/2

**Pull request merged**
